ClientApp/src/components/Artefact/Artefact.js

```diff
@@ -0,0 +1,265 @@
+import React from 'react'
+
+import Divider from '@material-ui/core/Divider'
+
+import './Artefact.css'
+
+// Presuming that bootstrap css and js have been imported from the parent page
+
+/* This page is intended to display details, comments and questions related to a particular artefact.
+*/
+
+// The styles attributes are grouped into this variable.
+const style ={
+    // style for the side section of the page
+    Paper:{padding:20,marginTop:10,marginBottom:10,marginLeft:10,marginRight:10},
+    
+    Card:{width:'500px',marginTop:30,marginBottom:30},
+    AppBar: {background:'blue'},
+    gridBelowDivider: {marginBottom:30,marginTop:20},
+    familyTreeButton: {margin:10},
+    familyTreeButtonIcon: {marginLeft:10},
+
+    // style for the number of artefacts and comment icons
+    icon:{marginRight:10},
+    
+    bio:{marginTop:20},
+    editImage: {
+        position: 'absolute',
+        transform:'translate(100%,-800%)',
+        display: 'block',
+    },
+    editName:{marginBottom:'30px'},
+}
+// The styles for the avatar (profile picture)
+const style2 = {
+    artefactFrame: {
+        margin: 10,
+        width: '80%',
+        height: '80%',
+        cursor:'pointer',
+    },
+}
+
+const { Component } = React
+
+const src = 'https://source.unsplash.com/random'
+
+class ImgBox extends Component {
+  
+  state = {
+    backgroundImage: `url(${src})`,
+    backgroundPosition: '0% 0%'
+  }
+  // allows the image to be enlarged while hovering the mouse cursor over the image
+  handleMouseMove = e => {
+    const { left, top, width, height } = e.target.getBoundingClientRect()
+    const x = (e.pageX - left) / width * 100
+    const y = (e.pageY - top) / height * 100
+    this.setState({ backgroundPosition: `${x}% ${y}%` })
+  }
+
+  render = () =>
+    <figure id="imgBoxFigure" onMouseMove={this.handleMouseMove} style={this.state}>
+      <img id="imgBoxImg" src={src} />
+    </figure>
+}
+
+// The main section of the Artefact page
+export default Artefact =>
+    <React.Fragment>
+
+        <div class="container">
```

Comment:
In React one uses `className=` in place of `class=` because `class` is a reserved keyword in Javascript.

Reply:
please allow this to merge since this will be resolved in the later commit